deploy/scripts/env.sh

```
@@ -11,3 +15,58 @@
+    if ! podman secret exists flightctl-postgresql-user-password; then
+        echo "Creating secret flightctl-postgresql-user-password"
+        if ! podman secret create --env flightctl-postgresql-user-password FLIGHTCTL_POSTGRESQL_USER_PASSWORD; then
+            echo "Error creating secret flightctl-postgresql-user-password"
+        fi
+    fi
+}
+
+ensure_kv_secrets() {
+    echo "Ensuring secrets for KV"
+    if [ -z "${FLIGHTCTL_KV_PASSWORD}" ]; then
+        export FLIGHTCTL_KV_PASSWORD=$(generate_password)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)